CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- [Demo] Fatal errors in the demo will now cause a fatal screen if `fatal=1` is present in the
```

Comment:
Actually can you remove this? We dont need CHANGELOG for demo change anymore.

Reply:
@devalevenkatesh  you can remove mine as well. `[Demo] Fix audio cannot be recorded issue`